crates/pallet-domains/src/staking.rs

```diff
@@ -325,11 +324,6 @@ pub fn do_register_operator<T: Config>(
             Error::InvalidOperatorSigningKey
         );
 
-        ensure!(
-            !OperatorSigningKey::<T>::contains_key(config.signing_key.clone()),
-            Error::DuplicateOperatorSigningKey
```

Comment:
we can remove this error as well

Reply:
That error is already removed here:
https://github.com/autonomys/subspace/pull/3265/files#diff-bc31bd26a15ac3e6de45a5242e5fc7883136b699fa411a472d0d3eb45c50f0f9L288